### PR TITLE
added additional packages needed to build QGC

### PR DESCRIPTION
**Pull request by barzanisar**
These packages are needed to solve the issue described in https://github.com/mavlink/qgroundcontrol/issues/6165

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ You **need to install Qt as described below** instead of using pre-built package
     * Windows: Make sure to install VS 2015 32 bit package.
 
 ###### Install additional packages:
-* Ubuntu: sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev
+* Ubuntu: sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev libgstreamer1.0-0 gstreamer1.0-plugins-base libgstreamer-plugins-base1.0-dev gstreamer1.0*
```
Comment (hamishwillee) [inline README.md]: I assume similar packages will not be needed for Fedora, archilinux, windows?

Comment (barzanisar) [reply README.md]: I didn't try on those. I tried in ubuntu which gave me some build errors and found out that these packages were also needed. If I later try with windows, I'll update this document accordingly.

Comment (hamishwillee) [reply README.md]: Works for me. Added @DonLakeFlyer as reviewer :-)

**Comment by dogmaphobic** [comment]
Thanks! This was long needed. Regarding Windows, only Linux has this issue as the packages are split into a zillion separate smaller packages with inter/extra dependencies. All other platforms are shipped as a single, monolithic set of libraries.

I would assume other Linux distributions would require equivalent work but the packages should be the same. I would imagine just the package manager would be different.

**Comment by DonLakeFlyer** [comment]
The packages for gstreamer were here: https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md. Not on the main README. Not sure having them be in two places (and different) is a good idea?

**Comment by DonLakeFlyer** [comment]
Also the set here https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md was working. Is this setup better?

**Comment by DonLakeFlyer** [comment]
These updated instructions fail on a clean up to date 16.04 image. Working on correct set. Then I'll revert this pull and update the streaming side readme. Something changed in latest 16.04 updates to break things from what was previously verified to work during stable 3.3 rollout.

**Pull request merged**